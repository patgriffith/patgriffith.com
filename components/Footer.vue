<template>
    <div>
        <EmailSignup />
        <footer class="bg-purple-800 mt-56">
            <p class="p-6 text-white text-center">Copyright @2020 Patrick Lorenzut</p>
        </footer>
        <TitleTwister />
        <ChatWidget />

    </div>
</template>
<script>
import EmailSignup from '~/components/EmailSignup.vue'
import TitleTwister from '~/components/TitleTwister.vue'
import ChatWidget from '~/components/ChatWidget.vue'
export default {
    components: {
        EmailSignup,
        TitleTwister,
        ChatWidget,
    }
}
</script>
