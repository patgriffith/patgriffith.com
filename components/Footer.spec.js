import { shallowMount } from '@vue/test-utils';
import jestConfig from '../jest.config';
import Footer from './Footer';

jest.mock('~/components/TitleTwister.vue', () => {})
jest.mock('~/components/ChatWidget.vue', () => {})

describe('<Footer />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(Footer);
  })

  it('should render <Footer/> without crashing', () => {
    expect(wrapper.element).toMatchSnapshot();
  })
})
