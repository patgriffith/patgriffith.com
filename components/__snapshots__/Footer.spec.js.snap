// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render <Footer/> without crashing 1`] = `
<div>
  <emailsignup-stub />
   
  <footer
    class="bg-purple-800 mt-56"
  >
    <p
      class="p-6 text-white text-center"
    >
      Copyright @2020 Patrick Lorenzut
    </p>
  </footer>
   
  <titletwister />
   
  <chatwidget />
</div>
`;
